docs/UsersGuide/source/BuildingRunningTesting/ContainerQuickstart.rst

```diff
@@ -271,9 +271,9 @@ The ``wflow_<platform>`` modulefile will then output instructions to activate th
 .. code-block:: console
 
    Please do the following to activate conda:
-       > conda activate workflow_tools
+       > conda activate srw_app
```

Comment:
@christinaholtNOAA We might want to leave this chapter as-is because your instructions will only work if/when the containers are updated, and I don't think we do that automatically yet.